<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>std::vector demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QListWidget" name="list_widget"/>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="1,2,2,1">
      <item row="6" column="2" colspan="2">
       <widget class="QPushButton" name="pb_plus">
        <property name="text">
         <string>++</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QLabel" name="lbl_4">
        <property name="text">
         <string>Итератор</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="pb_days">
        <property name="text">
         <string>= дни недели</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="pb_erase">
        <property name="text">
         <string>erase</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="pb_begin">
        <property name="text">
         <string>begin</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2" colspan="2">
       <widget class="QPushButton" name="pb_months">
        <property name="text">
         <string>= месяцы</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="lbl_1">
        <property name="text">
         <string>Методы</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="lbl_3">
        <property name="text">
         <string>size():</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="3">
       <widget class="QLabel" name="lbl_2">
        <property name="text">
         <string>Заготовки</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="pb_minus">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="pb_clear">
        <property name="text">
         <string>clear</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pb_pop_back">
        <property name="text">
         <string>pop_back</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QPushButton" name="pb_insert">
        <property name="text">
         <string>insert</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1" colspan="3">
       <widget class="QLineEdit" name="txt_size"/>
      </item>
      <item row="7" column="2" colspan="2">
       <widget class="QPushButton" name="pb_end">
        <property name="text">
         <string>end</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="pb_push_back">
        <property name="text">
         <string>push_back</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLineEdit" name="txt_elem_content"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pb_edit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
